rts/at/bridge/pom.xml

```diff
@@ -95,11 +95,18 @@
         </dependency>
 
         <dependency>
-            <groupId>org.jboss.logging</groupId>
-            <artifactId>jboss-logging</artifactId>
-            <version>${version.org.jboss.logging.jboss-logging}</version>
-            <scope>provided</scope>
-        </dependency>
+	      <groupId>org.jboss.logging</groupId>
```

Comment:
I'm not against this I would like just point that if we put the dependency as default for whole `at` module then all sub-modules will inherit that. It will force the dependency to modules where it's not necessary. For `at` it seems it's just `webservices` module. Still it's a place where such dependency is not needed.